src/main/java/org/fungover/haze/Main.java

```diff
@@ -78,15 +78,17 @@ public static String executeCommand(HazeDatabase hazeDatabase, List<String> inpu
         String command = inputList.get(0).toUpperCase();
 
         return switch (command) {
+            case "SET" -> hazeDatabase.set(inputList);
+            case "GET" -> hazeDatabase.get(inputList.get(1));
```

Comment:
```suggestion
            case "GET" -> hazeDatabase.get(inputList);
```

When GET is called without an argument we will get an index out of bounds exception, so maybe send in the whole list and then edit the set method to adjust for that.